speech/google/cloud/speech/_gax.py

```diff
@@ -104,9 +104,9 @@ def async_recognize(self, sample, language_code=None,
         audio = RecognitionAudio(content=sample.content,
                                  uri=sample.source_uri)
         api = self._gapic_api
-        response = api.async_recognize(config=config, audio=audio)
+        operation_future = api.async_recognize(config=config, audio=audio)
 
-        return Operation.from_pb(response, self)
+        return Operation.from_pb(operation_future.last_operation_data(), self)
```

Comment:
Ok, yeah there's an [issue](https://github.com/GoogleCloudPlatform/google-cloud-python/issues/2694) about adding retry as a whole.

I'm not sure if having it in GAPIC makes this easier or harder to do.